### DISCUSSION TITLE
Hello Community, I am seeking help to download Wails.

### DISCUSSION THREAD

**Discussion by tysonclyde** [wailsapp/wails#2386] [Q&A]
I have tried downloading it following the instructions on the page under Docs/Getting Started. Unfortunately it isn't recognized on my computer. I just need some help and am seeking some guidance.  
![Screenshot 2023-02-19 191515](https://user-images.githubusercontent.com/39357812/219984578-36eb77d9-92ee-4d1a-b2be-3003b6da9095.png)

**Comment by leaanthony** [comment]
https://wails.io/docs/guides/troubleshooting#the-wails-command-appears-to-be-missing

**Reply by tysonclyde** [reply]
Below are the Paths I have set in my Environment Variables. Am I doing something wrong here?

I apologize if I seem like an idiot. I am quite new to programming. I found this and I got really inspired to use it. But for the past three days I have been struggling to get it to work. I shall not give up, and will persevere.



![](https://user-images.githubusercontent.com/39357812/220093914-c32fc4a6-2e39-462e-a2af-b4ff10f60d6a.png)

**Reply by tysonclyde** [reply]
I got it working!!!
